components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -111,8 +111,9 @@ public static String getConfigFilePath(String fileName) {
 
     /**
      * writees the properties into a file
+     *
      * @param properties properties
-     * @param filePath filepath
+     * @param filePath   filepath

```

Comment:
This is added by idea code formatter. It's because it's aligning the variable names - properties, and filepath to be on the same indentation. Still you wanted to rever this?